frontend/src/main/scala/akkaviz/frontend/DOMGlobalScope.scala

```diff
@@ -15,4 +15,6 @@ object DOMGlobalScope extends js.GlobalScope {
 
   def str2ab(str: String): ArrayBuffer = js.native
 
+  def Rickshaw: js.Dynamic = js.native
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Method name does not match the regular expression '^([a-z][A-Za-z0-9]*|<<|>>>|>|==|!=|<|<=|>|>=|\||&|\^|\+|-|\*|\/|%|\|\||&&|\+\+|--|\+=|-=)$'](https://www.codacy.com/app/lustefaniak/akka-viz/file/2150900103/issues/source?bid=3121282&fileBranchId=3151611#l18)